commerce_coordinator/apps/commercetools/catalog_info/edx_utils.py

```diff
@@ -40,6 +40,29 @@ def is_edx_lms_order(order: CTOrder) -> bool:
     return len(get_edx_items(order)) >= 1
 
 
+def get_line_item_attribute(in_line_item, in_attribute_name):  # pragma no cover
```

Comment:
This utility isn't just related to edX products. This can be reused for all LOBs in CT in future. Hence this is better placed in `utils` file and not `edx_utils`. Please make the relevant import changes too

Reply:
Done